File: langgraph/data/equiv/66.py
def calculate_uppercase_ascii_sum(input_string):
    if input_string == "":
        return 0
    
    total = 0
    for character in input_string:
        if character.isupper():
            total += ord(character)
    
    return total
# method name: digitSum -> calculate_uppercase_ascii_sum
# parameter name: s -> input_string
# replaced the generator expression with a for loop
# introduced a 'total' variable to accumulate the sum
# used if statement inside the loop to check for uppercase characters

File: langgraph/result/qwen/3b-eq2js/66.js
function calculateUppercaseAsciiSum(inputString) {
    if (inputString === "") {
        return 0;
    }

    let total = 0;
    for (let i = 0; i < inputString.length; i++) {
        const character = inputString[i];
        if (character >= 'A' && character <= 'Z') {
            total += character.charCodeAt(0);
        }
    }

    return total;
}